src/Xamarin.Android.Build.Tasks/Tasks/CheckForInvalidResourceFileNames.cs

```diff
@@ -30,6 +30,10 @@ public override bool RunTask ()
 					resourceFile = resource.ItemSpec;
 				var fileName = Path.GetFileName (resourceFile);
 				var directory = Path.GetFileName (Path.GetDirectoryName (resourceFile));
+				char firstChar = fileName [0];
+				if (!(Char.IsLetter (firstChar) || firstChar != '_')) {
```

Comment:
I think maybe the logic is backwards checking against `_`, should it be?
```suggestion
				if (!Char.IsLetter (firstChar) && firstChar != '_') {
```

Reply:
Doh, I thought I'd created a draft PR 🤦  Good catch though :D